WalletWasabi/Models/SerializableException.cs

```diff
@@ -0,0 +1,50 @@
+using Newtonsoft.Json;
+using System;
+using System.Text;
+
+namespace WalletWasabi.Models
+{
+	[JsonObject(MemberSerialization.OptIn)]
+	public class SerializableException
```

Comment:
It would be nice to add a simple test for this class that shows how the exception serializes. I would expect it should be fairly easy.

Reply:
Fixed